vpr/src/server/gateio.h

```diff
@@ -150,7 +153,13 @@ class GateIO
     GateIO(GateIO&&) = delete;
     GateIO& operator=(GateIO&&) = delete;
 
-    // Check if the port listening process is currently running
+    /**
+    * @brief Checks if the port listening process is currently running.
+    *
+    * This method returns a boolean indicating whether the port listening process is currently running.
```

Comment:
Could just shorten first two lines to "Returns a bool indicating whether or not the port listening process is currently running."

Reply:
solved [b5521924d1ffd0efd7148fd3aae3d949fb05e9eb](https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/b5521924d1ffd0efd7148fd3aae3d949fb05e9eb)
(will be updated later)